app/config/collections.php

```diff
@@ -2810,6 +3460,27 @@
                 'lengths' => [128],
                 'orders' => [Database::ORDER_ASC],
             ],
+            [
+                '$id' => ID::custom('_key_agent'),
+                'type' => Database::INDEX_KEY,
+                'attributes' => ['agent'],
+                'lengths' => [2048],
+                'orders' => [Database::ORDER_ASC],
+            ],
+            [
+                '$id' => ID::custom('_key_path'),
+                'type' => Database::INDEX_KEY,
+                'attributes' => ['path'],
+                'lengths' => [2048],
+                'orders' => [Database::ORDER_ASC],
+            ],
+            [
+                '$id' => ID::custom('_key_method'),
+                'type' => Database::INDEX_KEY,
+                'attributes' => ['method'],
+                'lengths' => [128],
```

Comment:
No need to write the length, can be left empty, since the attribute size is 128 which is less than 768.